github/enterprise_manage_ghes.go

```diff
@@ -0,0 +1,163 @@
+// Copyright 2013 The go-github AUTHORS. All rights reserved.
```

Comment:
For all new files, please use the current year:

```suggestion
// Copyright 2025 The go-github AUTHORS. All rights reserved.
```